src/AllCountries.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useState } from "react";
+import Card from "./Card";
+import SidebarWidth from "./Sidebar";
+import { filterOptions } from "./ConstantData";
+import Filter from "./Filter";
+
+export default function AllCountries() {
+  const FetchCountriesData = async () => {
+    try {
+      const Url = "https://restcountries.com/v3.1/all";
+      let Response = await fetch(Url);
+      let ResponseData = Response.json();
+      return ResponseData;
+    } catch (error) {
+      throw new Error(error);
+    }
+  };
+  const [apiData, setapiData] = useState([]);
+  const [filterCountry, setfilterCountry] = useState(null);
+  const [showFilterCountries, setshowFilterCountries] = useState(false);
+  useEffect(() => {
+    (async () => {
+      let countryData = await FetchCountriesData();
+      let sliceData = countryData.slice(0, 20);
+
+      if (filterCountry == null) {
```

Comment:
We can use !fiterCountry instead of using == null.
!filterCountry will check for all falsy values like null, undefined, "", false, and 0